xmtp_mls/migrations/2024-08-07-213816_create-private-preference-store/up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE "private_preferences"(
```

Comment:
I wonder if we really want to call this "consent state".

Private preferences sounds more generic than what is really here. The schema is very specific to consent preferences (which is also the only kind we have right now).